ci-operator/step-registry/ipi/conf/aws/ipi-conf-aws-ref.yaml

```diff
@@ -1,6 +1,9 @@
 ref:
   as: ipi-conf-aws
-  from: base
+  from_image:
+    namespace: origin
+    name: centos
+    tag: '8'
```

Comment:
Can we use `tools` or similar to float with the product?

Reply:
Does that  always exist? sure

Reply:
Yeah, it's used by `oc debug node ...`